migrations/sql/V2022.03.28.15.26__create_nod_table.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS nod ( 
```

Comment:
Where possible we try to use the full words for table names so here 'NoticeOfDeparture' we ran in to to many issues with abbreviated names on MMS' database so we are trying to avoid that in the future.

Reply:
it is being referenced as `nod` even within JIRA tickets, so thought of keeping it consistent. But can easily make the change if that is convention.

Reply:
However, we are doing `now` for notice of work, so I don't know now, what is the right convention 😄

Reply:
This is resolved